cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -614,8 +614,16 @@ symlinkBuiltPackage :: Verbosity
 symlinkBuiltPackage verbosity overwritePolicy
                     mkSourceBinDir destDir
                     (pkg, components) =
-  traverse_ symlinkAndWarn exes
+  if null exes
+    then warn verbosity $ "You asked to install executables, "
```

Comment:
`warn` or `die'`?

Reply:
I'm for `warn`, since technically cabal succeeds in installing "all the exes of the package"